plugins/maximize/plugin.js

```diff
@@ -246,6 +297,21 @@
 						} );
 					}
 
+					// visualViewport is experimental feature so we need add a simple feature detection.
+					if ( window.visualViewport && checkIOS( container ) ) {
+						window.visualViewport.addEventListener( 'resize', function() {
+							if ( window.innerHeight !== savedHeight ) {
+								resizeHandler( window.innerHeight );
+
+								if ( savedHeight !== 0 && window.innerHeight > savedHeight ) {
```

Comment:
Could you add some comment with explanation why checking for `window.innerHeight > savedHeight` is used here?